ecs-cluster/keycloak.tf

```diff
@@ -327,9 +341,9 @@ resource "aws_ecs_service" "keycloak" {
   name                               = "${var.name}-service"
   cluster                            = aws_ecs_cluster.ecs.id
   task_definition                    = aws_ecs_task_definition.keycloak.arn
-  desired_count                      = 1
-  deployment_minimum_healthy_percent = 100
-  deployment_maximum_percent         = 200
+  desired_count                      = var.desired-count
+  deployment_minimum_healthy_percent = (var.desired-count < 1) ? 0 : 100
```

Comment:
Can we add a CloudWatch alarm for this? Maybe I should break out my cloudwatch insight notifier into a separate module

Reply:
Prob best to put it in a (public?) module and reuse it everywhere.